solutions/usaco-856.mdx

```diff
@@ -208,5 +208,45 @@ public class TheBucketList2 {
 ```
 	
 </JavaSection>
-
+<PySection>
+```py
+fin = open("blist.in", "r")
+
+max_time = 1001
```

Comment:
change this to 1000 and use `max_time + 1` where necessary

Reply:
there is literally no other use of max_time except for max_time + 1

Reply:
still doesn't make the actual value of `max_time` 1001